src/box/lua/console.c

```diff
@@ -983,6 +984,9 @@ tarantool_lua_console_init(struct lua_State *L)
 	console_hide_prompt_ref = luaL_ref(L, LUA_REGISTRYINDEX);
 	lua_pushcfunction(L, lbox_console_show_prompt);
 	console_show_prompt_ref = luaL_ref(L, LUA_REGISTRYINDEX);
+
+	VERIFY(luaT_dostring(L, "return function(t, k) return t[k] end") == 0);
```

Comment:
I'd drop a comment here to explain what does it verify and why.

Reply:
It's just a sanity check that the compilation is successful, not really meaningful. Anyways, dropped a comment here.